EnglishFilter/sections/specific.txt

```diff
@@ -261,6 +261,9 @@ its.porn##.preroll-blocker
 womennaked.net##.hrcht6
 jav789.com##span[id^="ad-"]
 ign.com##.zad.billboard
+claimcrypto.cc##a[href^="https://cryptowin.io"]
+shrtbtc.com,best18vd.com,idea4cash.com,bestmoneysites.com##td
```

Comment:
I guess you want to block leftovers, but this rule is too wide, especially when combined with other domains. You should create another rules for each site. But if you find a common pattern, add here.

Reply:
Hello! @zloyden I don't think it's necessary. EasyList Author, Ryan Brown, added this `idea4cash.com##td`. I just copied (hehe)>

Reply:
So it can be removed from PR as duplicate(and maybe another rules).

All other rules looks good. Will check on the sites.

Reply:
I will keep other domains except, `idea4cash.com`. Waiting for @zloyden

Reply:
Removed duplicate.

Reply:
@zloyden

Reply:
I've checked them and it looks ok. I have doubts about this `shrtbtc.com,best18vd.com,bestmoneysites.com##td`. Let's wait for @Alex-302 and @AdamWr.

Reply:
Okay. @zloyden But I am still confused, all domains added in `||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=` are also added to `#%#//scriptlet("abort-current-inline-script", "document.createElement", "ignielAdBlock")`. Check.

Reply:
>I have doubts about this `shrtbtc.com,best18vd.com,bestmoneysites.com##td`

I don't like rules like this, but if it doesn't break anything then I think that in this case it may be okay.

Reply:
It doesn't break anything confirmed.